Resources/config/routing.yml

```diff
@@ -1,3 +1,3 @@
 #kanbanize_client_index:
 #    path: /
-#    defaults: { _controller: ChapleanKanbanizeClientBundle:Default:index }
+#    defaults: { _controller: ChapleanKanbanizeApiClientBundle:Default:index }
```

Comment:
We can remove that, we don't need this file

Reply:
indeed :)